InSource/ProjectCreator.cs

```diff
@@ -180,6 +204,26 @@ private string GetProjectFolderPath(string name,string pathToTemplate)
 	        return string.Empty;
         }
 
+		// Validate if any language direction is configured on the custom template
+		private bool IsAnyLanguageDirection(string templatePath)
+		{
+			try
+			{
+				var templateXml = XElement.Load(templatePath);
+				var languageDirection = templateXml.Descendants("LanguageDirections").Descendants("LanguageDirection")?.FirstOrDefault();				
+				if (languageDirection == null)
```

Comment:
This condition and return can be refactored.

Reply:
Refactored the code: 
![image](https://user-images.githubusercontent.com/13149323/82440055-63543b00-9aa4-11ea-8aa3-5d63b2285842.png)